src/components/Map/Map/Map.tsx

```diff
@@ -31,11 +31,11 @@ type RegionArea = {
     features: GeoJSON.GeoJsonObject[];
 };
 
-interface Props extends Omit<GoogleMap, 'onLoad' | 'mapContainerStyle' | 'options'> {
+interface Props {
     /** The default parameters to determine the viewport when no circular or point markers are present. */
-    defaultArea:
+    defaultArea?:
         | {
-              address: string;
+              address: string | string[] | undefined;
```

Comment:
:+1:

Reply:
Because in LocationSelectorDialog.tsx  use     
```
function getDefaultArea() {
        if (initialMapAddress || country) {
            return { address: initialMapAddress || country };
        }

        return undefined;
    }
```
![image](https://github.com/textkernel/oneui/assets/104063210/b09e7c83-0662-4dbc-a042-dd980d2adca9)